model/connection.js

```diff
@@ -37,6 +37,40 @@ class Connection {
     sinks;
 
 
+    /**
+     * The abstract Connection this Connection Feature represents.
+     *
+     * @since 1.0.0
+     * @access public
+     *
+     * @type {object}
+     */
+    connection;
+
+    /**
+     * The segments that make up this Connection Feature.
+     *
+     * @since 1.0.0
+     * @access public
+     *
+     * @type {Array}
+     */
+    connSegments;
+
+    /**
+     * The type of connection.
+     *
+     * Currently always set to "channel". Treat this as a constant and do not
+     * change it.
+     *
+     * @since 1.0.0
+     * @access public
+     *
+     * @type {string}
+     */
+    connectionType;
```

Comment:
It doesn't really matter since there is only one type, but I believe the type is specified in the connection feature so you might want to put it there.

Reply:
I'm not sure I follow. Are saying I should set it to ```"channel"``` here? If so, I set it in the constructor to ```Validation.DEFAULT_CON_TYPE``` which currently equals ```"channel"```.

Reply:
What I mean is that in the ParchMint file I believe that the `type: channel` is in the feature portion and not in the connection portion, so would it make more sense to save that information in the feature rather than abstract connection?

Reply:
It's been moved to the ```ConnectionSegment``` class.